docker/README.md

```diff
@@ -0,0 +1,224 @@
+# Ruqola Setup on WSL or WINDOWS
```

Comment:
We  need to add license info see "README.md.license"

Use "reuse lint" command.
As we don't use kde CI support we can't see it but when we will merge in kde repo we will have some error.